docs/api/en/animation/tracks/BooleanKeyframeTrack.html

```diff
@@ -23,14 +23,18 @@ <h3>
 			[page:Array times] - (required) array of keyframe times.<br />
 			[page:Array values] - values for the keyframes at the times specified.<br />
 		</p>
+		<p>
+			This keyframe track type has no interpolation parameter because the
+			interpolation is always [page:Animation InterpolateDiscrete].
```

Comment:
If we want to make the API documentation technically correct, `BooleanKeyframeTrack` should also document the `interpolation` ctor parameter. The class does not overwrite the ctor so it inherits the same signature as `KeyframeTrack`.

We can state in the description which interpolation types are allowed.